test/checks/media/no-autoplay-audio.js

```diff
@@ -1,119 +1,126 @@
-  var checkSetup = axe.testUtils.checkSetup;
-  var checkContext = axe.testUtils.MockCheckContext();
-  var preloadOptions = { preload: { assets: ['media'] } };
-
-  before(function () {
-    check = checks['no-autoplay-audio'];
-  });
-
-  afterEach(function () {
-    fixture.innerHTML = '';
-    axe._tree = undefined;
+  afterEach(() => {
     checkContext.reset();
   });
 
-  it('returns undefined when <audio> has no source (duration cannot be interpreted)', function (done) {
-    var checkArgs = checkSetup('<audio id="target"></audio>');
-    axe.utils.preload(preloadOptions).then(function () {
-      assert.isUndefined(check.evaluate.apply(checkContext, checkArgs));
-      done();
-    });
+  it('returns undefined when <audio> has no source (duration cannot be interpreted)', async () => {
+    const checkArgs = checkSetup('<audio id="target"></audio>');
+    await axe.utils.preload(preloadOptions);
+    assert.isUndefined(check.evaluate.apply(checkContext, checkArgs));
   });
 
-  it('returns undefined when <video> has no source (duration cannot be interpreted)', function (done) {
-    var checkArgs = checkSetup('<video id="target"><source src=""/></video>');
-    axe.utils.preload(preloadOptions).then(function () {
-      assert.isUndefined(check.evaluate.apply(checkContext, checkArgs));
-      done();
-    });
+  it('returns undefined when <video> has no source (duration cannot be interpreted)', async () => {
+    const checkArgs = checkSetup('<video id="target"><source src=""/></video>');
+    await axe.utils.preload(preloadOptions);
+    assert.isUndefined(check.evaluate.apply(checkContext, checkArgs));
   });
 
-  it('returns false when <audio> can autoplay and has no controls mechanism', function (done) {
-    var checkArgs = checkSetup(
+  it('returns false when <audio> can autoplay and has no controls mechanism', async () => {
+    const checkArgs = checkSetup(
       '<audio id="target" src="/test/assets/moon-speech.mp3" autoplay="true"></audio>'
     );
-    axe.utils.preload(preloadOptions).then(function () {
-      assert.isFalse(check.evaluate.apply(checkContext, checkArgs));
-      done();
-    });
+    await axe.utils.preload(preloadOptions);
+    assert.isFalse(check.evaluate.apply(checkContext, checkArgs));
   });
 
-  it('returns false when <video> can autoplay and has no controls mechanism', function (done) {
-    var checkArgs = checkSetup(
-      '<video id="target" autoplay="true">' +
-        '<source src="/test/assets/video.webm" type="video/webm" />' +
-        '<source src="/test/assets/video.mp4" type="video/mp4" />' +
-        '</video>'
-    );
-    axe.utils.preload(preloadOptions).then(function () {
-      assert.isFalse(check.evaluate.apply(checkContext, checkArgs));
-      done();
-    });
+  it('returns false when <video> can autoplay and has no controls mechanism', async () => {
+    const checkArgs = checkSetup(`
+      <video id="target" autoplay="true">
```

Comment:
why are we explicitly specifying the value of the autoplay? it's it just a boolean attribute? E.g. `autoplay = "false"` is equal to `autoplay="true"` and will still autoplay.

Reply:
No reason other than just copy paste from previous tests.